internal/venice-common/src/test/java/com/linkedin/venice/pubsub/PubSubUtilTest.java

```diff
@@ -243,4 +248,80 @@ public void testCalculateSeekOffset() {
     // Case 2: Inclusive = false, should return base offset + 1
     assertEquals(PubSubUtil.calculateSeekOffset(100L, false), 101L, "Exclusive seek should return base offset + 1");
   }
+
+  @Test
+  public void testGetBase64EncodedString() {
+    // Test with normal byte array
+    byte[] testBytes = new ApacheKafkaOffsetPosition(12345L).toWireFormatBytes();
+    String encoded = PubSubUtil.getBase64EncodedString(testBytes);
```

Comment:
We'll only encode actual position bytes and not the whole wire format which includes type + position raw bytes